archive/bagger/src/bagger_processor.py

```diff
@@ -1,23 +1,38 @@
+import time
+
 import logging
 import traceback
+
 import bagger
-import time
+import notifier
 
 
 def process_bagging_message(message):
     identifier = message.get("identifier", None)
     do_not_bag = message.get("do_not_bag", True)
-    result = {"identifier": identifier, "duration": -1, "error": None}
+
+    result = {
+        "identifier": identifier,
+        "upload_location": None,
+        "duration": -1,
+        "error": None,
+    }
+
     if identifier is not None:
         start = time.time()
         try:
-            bagger.bag_from_identifier(identifier, do_not_bag)
-        except Exception:
+            result["upload_location"] = bagger.bag_from_identifier(
+                identifier, do_not_bag
+            )
+            end = time.time()
+            result["duration"] = end - start
+
+            notifier.bagging_complete(result)
+
+        except:
```

Comment:
The linter is whinging about the bare `except`, which is almost never what you want. A bare `except:` will also catch things like `SystemExit` and `KeyboardInterrupt` – it’s unusual to want to catch those low-level exceptions!

```suggestion
        except Exception:
```